third_party/2and3/mypy_extensions.pyi

```diff
@@ -9,3 +9,5 @@ def TypedDict(typename: str, fields: Dict[str, Type[_T]]) -> Type[dict]: ...
 # This type is equivalent to the None type, but the no-op Union is necessary to
 # distinguish the None type from the None value.
 NoReturn = Union[None]
+
+def decorated_type(t: Type[Any]) -> Callable[[_T], _T]: ...
```

Comment:
The argument type should probably be `Any`, since some valid types aren't real types (e.g. `None`).